bayesian/api/api_v2.py

```diff
@@ -326,7 +326,7 @@ def stack_report_url(stack_id):
             "ver": "v3",
             "uuid": crda_key
         },
-        "user_key": THREESCALE_ACCOUNT_SECRET
+        "user_key": THREESCALE_ACCOUNT_VALUE
```

Comment:
Use consistent naming. It should be called as  THREESCALE_USER_KEY

Reply:
Ack